internal/databases/postgres/backup_list_handler.go

```diff
@@ -25,52 +24,35 @@ func HandleDetailedBackupList(folder storage.Folder, pretty bool, json bool) {
 
 	backupDetails, err := GetBackupsDetails(folder, backups)
 	tracelog.ErrorLogger.FatalOnError(err)
-
+	SortBackupDetails(backupDetails)
+	
 	switch {
 	case json:
 		err = internal.WriteAsJSON(backupDetails, os.Stdout, pretty)
 	case pretty:
-		writePrettyBackupListDetails(backupDetails, os.Stdout)
+		WritePrettyBackupListDetails(backupDetails, os.Stdout)
 	default:
-		err = writeBackupListDetails(backupDetails, os.Stdout)
+		err = WriteBackupListDetails(backupDetails, os.Stdout)
 	}
 	tracelog.ErrorLogger.FatalOnError(err)
 }
 
-func GetBackupsDetails(folder storage.Folder, backups []internal.BackupTime) ([]BackupDetail, error) {
-	backupsDetails := make([]BackupDetail, 0, len(backups))
-	for i := len(backups) - 1; i >= 0; i-- {
-		details, err := GetBackupDetails(folder, backups[i])
-		if err != nil {
-			return nil, err
-		}
-		backupsDetails = append(backupsDetails, details)
-	}
-	return backupsDetails, nil
-}
-
-func GetBackupDetails(folder storage.Folder, backupTime internal.BackupTime) (BackupDetail, error) {
-	backup := NewBackup(folder, backupTime.BackupName)
-
-	metaData, err := backup.FetchMeta()
-	if err != nil {
-		return BackupDetail{}, err
-	}
-	return BackupDetail{backupTime, metaData}, nil
-}
-
 // TODO : unit tests
-func writeBackupListDetails(backupDetails []BackupDetail, output io.Writer) error {
+func WriteBackupListDetails(backupDetails []BackupDetail, output io.Writer) error {
 	writer := tabwriter.NewWriter(output, 0, 0, 1, ' ', 0)
 	defer writer.Flush()
-	_, err := fmt.Fprintln(writer, "name\tlast_modified\twal_segment_backup_start\tstart_time\tfinish_time\thostname\tdata_dir\tpg_version\tstart_lsn\tfinish_lsn\tis_permanent") //nolint:lll
```

Comment:
better leave the linter hint: //nolint:lll, it's easier to read the table header when it is one-liner